content/id/docs/reference/kubectl/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: "kubectl CLI"
```

Comment:
Wondering if we're using CLI or command line interface

Reply:
actually we can translate this, how about "baris perintah kubectl"

Reply:
fixed in e052cc3